### PR TITLE
Repeated n experiment

### PR DESCRIPTION
**Pull request by tsor13**

### CODE REVIEW

File: coding/experiments/nyt/repeated_n_all_different/experiment.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+templatizer = Templatizer(dataset_name='nytimes')
+# do for 10 runs
+n_runs = 10
+n_examples = 31
+
+for run in range(n_runs):
+    for n in range(31):
+        # get today's date as string MM-DD-YYYY
+        today = pd.to_datetime('today').strftime('%m-%d-%Y')
+        exp_dir = os.path.join(f'experiments/nyt/repeated_n_all_different/{today}/fewshot/{n}/run{run}')
```
Comment (chrisrytting) [inline coding/experiments/nyt/repeated_n_all_different/experiment.py]: I'd like `coding/experiments/nyt/repeated_n_all_different/date/result` to be   `coding/experiments/nyt/date/repeated_n_all_different/result`

**Review by chrisrytting — CHANGES_REQUESTED** [review]
Great work but does this make sense?

File: coding/experiments/nyt/repeated_n_all_different/experiment.py (hunk 1 above)
Comment (tsor13) [reply coding/experiments/nyt/repeated_n_all_different/experiment.py]: Done!

**Review by chrisrytting — APPROVED** [review]
Great!

**Pull request closed without merging**